static/js/shared/util.ts

```diff
@@ -94,8 +94,8 @@ export function isDateTooFar(date: string): boolean {
 }
 
 export function getCappedStatVarDate(statVar: string): string {
-  // Only want to cap stat var date for stat vars with RCP.
-  if (!statVar.includes("_RCP")) {
+  // Only want to cap stat var date for stat vars with RCP or SSP.
+  if (!statVar.includes("_RCP") && !statVar.includes("_SSP")) {
     return "";
   }
   for (const svSubstring of NO_DATE_CAP_RCP_STATVARS) {
```

Comment:
Could you remind me why this is not "NumberOfMonths_5CelsiusOrMore_", but "NumberOfMonths_5CelsiusOrMore_Percentile10AcrossModels_" ?

We have Percentile10, Percentile90 and Median...

Reply:
Looking through our chat history, it looks like it was because "That class of stat-vars includes decadal aggregates and are available for dates 2030, 2050 and 2100.  Now the "percentile90" and "median" have data for 2030 and 2050.  But "percentile10" only has data for 2100."

Reply:
I see, so for percentile90 and median we want to show 2050. So maybe we can add that comment to the list?

Reply:
done